extensions/amp-analytics/amp-analytics.md

```diff
@@ -0,0 +1,97 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+**This extension is still a work in progress. Details below can change.**
+
+### <a name="amp-analytics"></a> `amp-analytics`
+
+A generic framework to collect and send analytics data to multiple analytics vendors.
+
+#### <a name="behavior"></a>Behavior
+
+`<amp-analytics>` tag can be used to define events of interest, collect analytics data and then send it off to any URL.
+
+This tag provides built-in support for most widely used analytics services. In addition, the tag allows for the analytics config to be specified such that the analytics data can be sent to any specified URL is a custom format.
+
+The tag is driven by JSON config that can be specified in one of the three places: 
+
+**Remote configs**: This config is loaded in the page via a URL. The URL can be specified in the `config` attribute of the `amp-analytics` tag.
+**Inline configs**: This config is specified directly on the AMP page inside the `amp-analytics` tag.  
+**Built in configs**: The AMP platform has these configs built into the platform. These can be loaded onto the page via the `type` attribute on the `amp-analytics` tag.
+
+If more than one methods of specifying the config are used, the configs are merged together in the order above (highest to lowest precedence). 
+
+
+##### <a name="inline"></a>Inline Config with a custom host & url format 
+```javascript
+<amp-analytics>
+{
+  "host": "my-analytics.com:8080",
+  "requests": {
+    "base_hit": "/collect?v=1&_v=a0&aip=true&_s=HIT_COUNT&dl=DOMAIN&dt=TITLE&sr=SCREEN_WIDTHxSCREEN_HEIGHT&ht=TIMESTAMP&cid=CLIENT_IDENTIFIER&tid=ACCOUNT"
```

Comment:
Link https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md somewhere in this doc.